### PR TITLE
fix: rectangle subtract func

### PR DESCRIPTION
**Pull request by yuhongz**
close #xxx, #yyy, #zzzz

### CODE REVIEW

File: packages/core/src/shared/__test__/rectangle.spec.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { describe, expect, it } from 'vitest';
+
+import { Rectangle } from '../rectangle';
+
+describe('test rectangle', () => {
```
Comment (wzhudev) [inline packages/core/src/shared/__test__/rectangle.spec.ts:21]:
```suggestion
describe('test "Rectangle"', () => {
```

Please use double quotes to surrond the API you're about to test.

File: packages/core/src/shared/__test__/rectangle.spec.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    it('test subtract', () => {
```
Comment (wzhudev) [inline packages/core/src/shared/__test__/rectangle.spec.ts:22]:
```suggestion
    it('test "subtract"', () => {
```

**Review by wzhudev — COMMENTED** [review]
LGTM

**Review by wzhudev — APPROVED** [review]
LGTM

**Review by DR-Univer — APPROVED** [review]

**Pull request merged**